test/e2e/tests/multiple-transactions.spec.js

```diff
@@ -95,4 +101,125 @@ describe('Confirm transactions', function () {
       },
     );
   });
+
+  it('creates multiple queued transactions, then confirms', async function () {
```

Comment:
The existing tests appear to do the same but with a fixture to provide the transactions rather than end to end.

For the sake of time and maintainability, should we delete the previous ones if these encompass them?

Reply:
100%, I've removed the test using fixtures as the new ones are covering end to end.